modules/ROOT/pages/tutorial/tutorial-clustering-docker.adoc

```diff
@@ -271,7 +271,7 @@ The following environment variables are specific to the Neo4j cluster, and are a
 
 * `+NEO4J_initial_server_mode__constraint+`: the database mode, defaults to `NONE`, can be set to `PRIMARY` or `SECONDARY`.
 * `+NEO4J_dbms_cluster_endpoints+`: a comma-separated list of endpoints, which a server should contact to discover other cluster servers.
-* `+NEO4J_server.cluster.advertised_address+`: hostname/IP address and port to advertise for transaction handling and v2 discovery.
+* `+NEO4J_server.cluster.advertised_address+`: hostname/IP address and port to advertise for transaction handling and discovery protocol.
```

Comment:
protocol or service?

Reply:
🧐 I think we use both words, but if it's confusing, let's be consistent - I'll change it to 'service'